src/components/SelectionList/Search/TransactionListItemRow.tsx

```diff
@@ -76,6 +77,7 @@ function ReceiptCell({transactionItem}: TransactionCellProps) {
     const StyleUtils = useStyleUtils();
 
     const backgroundStyles = transactionItem.isSelected ? StyleUtils.getBackgroundColorStyle(theme.buttonHoveredBG) : StyleUtils.getBackgroundColorStyle(theme.border);
+    const canModifyReceipt = !ReportUtils.isReportApproved(transactionItem.reportID) && !ReportUtils.isSettled(transactionItem.reportID) && !ReportUtils.isClosedReport(transactionItem.reportID);
```

Comment:
Instead of checking if the report is not eligible we should check if the report is eligible i.e. canModifyReceipt = isOpen || isSubmitted